app/api/speakers_calls.py

```diff
@@ -29,21 +29,21 @@ class Meta:
     hash = fields.Str()
     privacy = fields.String()
     event = Relationship(attribute='event',
-                         self_view='v1.call_for_paper_event',
+                         self_view='v1.speakers_calls_event',
                          self_view_kwargs={'id': '<id>'},
                          related_view='v1.event_detail',
-                         related_view_kwargs={'call_for_paper_id': '<id>'},
+                         related_view_kwargs={'speakers_calls_id': '<id>'},
                          schema='EventSchema',
                          type_='event')
 
 
-class CallForPaperList(ResourceList):
+class SpeakersCallsList(ResourceList):
```

Comment:
SpeakersCallList? It wont make much a difference but maybe keep it singular like rest of the api?